Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -133,12 +133,23 @@ YUI.add('ez-relation-editview', function (Y) {
          * @param {EventFacade} e
          */
         _runUniversalDiscovery: function (e) {
+            var that = this;
```

Comment:
This is copied from the relationlist's file: https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/fields/ez-relationlist-editview.js#L184

I can rename this, but then it wont be consistent.